server/run.py

```diff
@@ -0,0 +1,47 @@
+# to avoid circular import, must save with import views, models, resources at bottom
+# to save without auto-formatting on vs code, `CTRL + K + SHIFT + S`
+
+from flask import Flask
+from flask_restful import Api
+from flask_sqlalchemy import SQLAlchemy
+from flask_jwt_extended import JWTManager
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
```

Comment:
I don't think we need this actually if we are using Pipfile as it may load the dotenv automatically into environment varialbes when you are in the shell. FYI, you can get into the shell by typing `pipenv shell` or by running a specific command in the shell like this `pipenv run ....`